src/main/java/org/jenkins/tools/test/model/plugin_metadata/LegacyPluginMetadataExtractor.java

```diff
@@ -30,7 +30,7 @@ public Plugin extractMetadata(String pluginId, Manifest manifest, Model model) t
                 .withGitHash(model.getScm().getTag())
                 .withTag(model.getScm().getTag())
                 // Any multi-module projects have already been handled by now or require new hooks
-                .withModule(null)
+                .withModule(model.getArtifactId())
```

Comment:
this would be is incorrect and should be

```suggestion
                .withModule(":" + model.getArtifactId())
```